### DISCUSSION TITLE
Azure Pipelines: windows-2016 is deprecated

### DISCUSSION THREAD

**Discussion by Geod24** [dlang/dmd#13244] [CI]
https://github.com/actions/virtual-environments/issues/4312

Already visible in Azure Pipelines:
> ##[warning]The windows-2016 environment will be deprecated on November 15, 2021, and removed on March 15, 2022. Migrate to windows-latest instead. For more details see https://github.com/actions/virtual-environments/issues/4312

Brownouts:
> December 1, 2021 4:00pm UTC – 10:00pm UTC
> February 7, 2022 4:00pm UTC – 10:00pm UTC

**Comment by MoonlightSentinel** [comment]
The deprecated image can be replaced with `windows2019`, which also switches to Visual Studio 2019:

- #13482 
- dlang/druntime#3660
- dlang/phobos#8348

**Reply by PetarKirov** [reply]
@Geod24 pls force merge #13482

**Reply by Geod24** [reply]
Thanks for taking care of this!
